packager/hls/base/simple_hls_notifier_unittest.cc

```diff
@@ -645,6 +650,43 @@ TEST_F(SimpleHlsNotifierTest, EncryptionScheme) {
       stream_id, key_id, common_system_id_, iv, dummy_pssh_data));
 }
 
+// Verify that the Fairplay systemID is correctly handled when constructing
+// encryption info
+TEST_F(SimpleHlsNotifierTest, NotifyEncryptionUpdateFairplay) {
+  // Pointer released by SimpleHlsNotifier.
+  MockMediaPlaylist* mock_media_playlist =
+      new MockMediaPlaylist(kLivePlaylist, "playlist.m3u8", "", "");
+  SimpleHlsNotifier notifier(kLivePlaylist, kTestTimeShiftBufferDepth,
+                             kTestPrefix, kAnyOutputDir, kMasterPlaylistName);
+  const uint32_t stream_id =
+      SetupStream(kSampleAesProtectionScheme, mock_media_playlist, &notifier);
+  const std::vector<uint8_t> key_id(16, 0x00);
+  const std::vector<uint8_t> iv(16, 0x45);
+  std::vector<uint8_t> pssh_box = { 0x00, 0x00, 0x00, 0x34, 0x70, 0x73, 0x73,
+                                    0x68, 0x01, 0x00, 0x00, 0x00, 0x29, 0x70,
+                                    0x1F, 0xE4, 0x3C, 0xC7, 0x4A, 0x34, 0x8C,
+                                    0x5B, 0xAE, 0x90, 0xC7, 0x43, 0x9A, 0x47,
+                                    0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00,
+                                    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+                                    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+                                    0x00, 0x00, 0x00 };
```

Comment:
I don't think it matters. You may replace it with a dummy value, e.g.

const std::vector<uint8_t> dummy_pssh_data(10, 'p');